gradio/blocks.py

```diff
@@ -586,6 +605,8 @@ def get_block_instance(id: int) -> Block:
             block_config["props"].pop("type", None)
             block_config["props"].pop("name", None)
             style = block_config["props"].pop("style", None)
+            if block_config["props"].get("root_url") is None:
```

Comment:
If the Space is already loading from another Space, it should use the root_url of the parent Space. This allows loading a chain of Spaces